tinygrad/runtime/ops_webgl.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import functools
+from tinygrad.helpers import dtypes, DType
+from tinygrad.device import Compiled, Allocator
+from tinygrad.codegen.kernel import LinearizerOptions
+from tinygrad.renderer.cstyle import uops_to_cstyle
+from tinygrad.renderer.glsl import GLSLLanguage
+import moderngl
+import os
+
+CI = os.getenv("CI", "") != ""
+ctx = moderngl.create_standalone_context()
+dtype_map = { dtypes.float64: "f8", dtypes.float: "f4", dtypes.half: "f2", dtypes.int32: "i4", dtypes.uint32: "u4", dtypes.bool: "i1"}
+class WebGLProgram:
+  def __init__(self, name: str, prg: str, bufs:int=0, vars:int=0): self.name, self.prg = name, ctx.program(vertex_shader="#version 330\nprecision highp float;\nin vec2 in_position;in vec2 in_uv;out vec2 uv;void main(){gl_Position=vec4(in_position,0.0,1.0);uv=in_uv;}", fragment_shader=prg)
+  def __call__(self, *bufs, global_size, local_size, wait=False):
+    vert, uv =ctx.buffer(np.asarray([-1, 1, -1, -1, 1, 1, 1, -1], dtype='f4').tobytes()), ctx.buffer(np.asarray([0, 1, 0, 0, 1, 1, 1, 0], dtype='f4').tobytes())
+    self.vao = ctx.vertex_array(self.prg, [])
+    self.vao.bind(0 if CI else self.prg["in_position"].location, buffer=vert, cls='f', fmt='2f4')
```

Comment:
In CI `self.prg["in_position"].location` and `self.prg["in_uv"].location`  fail with `KeyError`, for some reason it can't look up the uniform by key, which seems to be a bug in `moderngl`, this is why we bind hardcoded uniform positions.